src/app/components/elements/TestimonialComment.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+interface TestimonialCommentProps {
+  imageSrc: string;
+  altText: string;
+  text: string;
+}
+
+const TestimonialComment: React.FC<TestimonialCommentProps> = ({ imageSrc, altText, text }) => (
+  <div className="resources-center-container">
+    <div className="resources-comment">
+      <div className="resources-comment-content">
```

Comment:
Could all the `div` classes under resources be refactored using bootstrap instead?

Reply:
That is a good suggestion